packages/html-handler/src/utils.ts

```diff
@@ -1,13 +1,25 @@
 import * as crypto from 'crypto';
-import { LogEntry, LogEntryQueue } from './types';
+import { v4 } from 'node-uuid';
 
-export function findDefiningEntry(queue: LogEntryQueue): LogEntry {
+import { LogEntry, TContextBase, TContextShape } from '@debugr/core';
+import { LogEntryQueue } from './types';
+
+export function findDefiningEntry<
+  TTaskContext extends TContextBase = TContextBase,
+  TGlobalContext extends TContextShape = {},
+>(
+  queue: LogEntryQueue<Partial<TTaskContext>, TGlobalContext>,
+): LogEntry<Partial<TTaskContext>, TGlobalContext> {
   if (queue.firstOverThreshold !== undefined) {
     return queue.entries[queue.firstOverThreshold];
   } else if (!queue.entries.length) {
     return {
-      ts: 0,
+      ts: new Date(0),
       level: 4,
+      // @ts-ignore
+      context: {
+        processId: v4(),
+      },
       message: 'EMPTY QUEUE!',
       data: queue,
     };
```

Comment:
nebylo by lepší aby tady `level` použil nějakou konstantu? & proč je tam to `@ts-ignore`, neměl by resolvnutej `context` vždycky definovat `processId`?

Reply:
level je konstanta. @ts-ignore je kvuli demenci typescriptu....